plugins/task-plugin/src/variable/project-path-variable-resolver.ts

```diff
@@ -38,33 +38,7 @@ export class ProjectPathVariableResolver {
             return this.onError('Projects root is not set');
         }
 
-        const selections = await theia.commands.executeCommand<Uri[]>(SELECTED_CONTEXT_COMMAND);
```

Comment:
Maybe we should not remove this logic if at the moment it's just deprecated? 
For example, user starts a workspace from devfile with two defined projects: `theia` and `che-theia`.
By default user has `/projects` as workspace folder.
According to these changes  `${current.project.path}` returns `/projects`, not  `/projects/theia` for example.
Isn't it?
Please correct me if I'm wrong.

Reply:
This command relies on the current selection. When workspace opens at first time, the selection is not initialized and we're getting error message `... bad substitution ...`. It will return `/project` only when user opened explorer part or some file in editor.

Reply:
`selections` in case of opening workspace at first time will be `undefined`. So it useless and calling `${workspaceFolder}` will return the same result if editor or explorer is opened.

Reply:
I understand that there are bugs related to using this variable if user has incorrect selection, that's why we decided to use `${workspaceFolder}` instead of `${current.project.path}`.
But if user or our samples rely on `${current.project.path}`, these places will be broken after your changes!
isn't is?

Reply:
Our samples don't use this macro. If user uses this macro in own devfile, it will broken anyway and command won't execute. There will be created another PR into docs that describes, that fact, that this macro is deprecated and has to be replaced with another one. So, yes, this changes proposal changes behavior of the flow for this macro.

Reply:
steps to reproduce:
- start a workspace from a devfile, get project structure like:
![pr_structure](https://user-images.githubusercontent.com/5676062/70705091-0c1f9280-1cdc-11ea-8857-2bb91b929b2d.png)
- run the task:
```
{
                "type": "che",
                "label": "test2",
                "command": "sleep 3 && echo ${workspaceFolder}",
                "target": {
                    "workingDir": "/projects"
                }
            },
```
output: `/projects`
- select `che-theia` project and run the task:
```
{
                "type": "che",
                "label": "test2",
                "command": "sleep 3 && echo ${current.project.path}",
                "target": {
                    "workingDir": "/projects"
                }
            }
```
output: `/projects/che-theia`